operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2487,6 +2487,44 @@ nginx:
         }
       }
 
+# Use either this ingress or the gateway, but not both at once.
+# If you enable this, make sure to disable the gateway.
+# You'll need to supply authn configuration for your ingress controller.
+ingress:
+  enabled: false
+#  ingressClassName: nginx
+  annotations: {}
+#    nginx.ingress.kubernetes.io/auth-type: basic
+#    nginx.ingress.kubernetes.io/auth-secret: mimir-distributed-basic-auth
+#    nginx.ingress.kubernetes.io/auth-secret-type: auth-map
+#    nginx.ingress.kubernetes.io/configuration-snippet: |
+#      proxy_set_header X-Scope-OrgID $remote_user;
+  paths:
+    distributor-headless:
+      - /distributor
+      - /api/v1/push
+      - /otlp/v1/metrics
+    alertmanager-headless:
+      - /alertmanager
+      - /multitenant_alertmanager/status
+      - /api/v1/alerts
+    ruler:
+      - /prometheus/config/v1/rules
+      - /prometheus/api/v1/rules
+      - /prometheus/api/v1/alerts
+      - /ruler/ring
```

Comment:
i don't think this should be exposed to the outside world. I see it's exposed in the nginx config and I think that's wrong. I'll open a PR to remove it from there too